geoips/plugins/modules/readers/generic_satpy.py

```diff
@@ -0,0 +1,134 @@
+# # # This source code is protected under the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+
+"""Generic Satpy reader.
+
+Provides a GeoIPS wrapper for Satpy reader functionality. For more detailed
+documentation of Satpy, see the documentation here:
+https://satpy.readthedocs.io/en/stable/. This reader creates a Scene object with
+the provided filenames and specified reader, then loads provided channels. (The
+satpy_reader, and fnames arguments are passed to the Scene constructor)  The
```

Comment:
The parentheses portion is its own sentence but is intended to comment on the previous sentence to indicate how these arguments will be used.  This can be rewritten to be more clear.